blockchain/types/transaction_signing.go

```diff
@@ -542,6 +543,7 @@ func (s eip2930Signer) SignatureValues(tx *Transaction, sig []byte) (R, S, V *bi
 
 	// Check that chain ID of tx matches the signer. We also accept ID zero or nil here,
 	// because it indicates that the chain ID was not specified in the tx.
+	// NOTE: Kaia allow chain ID to be nil in this fork
```

Comment:
```suggestion
	// NOTE: Kaia doesn't allow chain ID to be nil in this fork
```

Reply:
Sorry, I've misunderstood once. The blow condition can allow chain ID to be nil